[2019-04-12 17:23:29,555] - log_print.py] - ERROR: 1 .实名认证:http://116.62.67.80:9050/baseVerify/faceAuthentication  超时响应,请注意!!
[2019-04-12 17:28:51,469] - log_print.py] - ERROR: 1 .实名认证:http://116.62.67.80:9050/baseVerify/faceAuthentication  超时响应,请注意!!
[2019-04-12 17:28:59,122] - log_print.py] - ERROR: 2 .芝麻分认证:http://116.62.67.80:9050/riskVerify/zhiMaAuth  超时响应,请注意!!
[2019-04-12 17:29:06,259] - log_print.py] - ERROR: 3 .运营商认证:http://116.62.67.80:9050/tjVerify/operatorVerify  超时响应,请注意!!
[2019-04-12 17:29:13,454] - log_print.py] - ERROR: 4 .身份证纠错:http://116.62.67.80:9050/baseVerify/faceErrorCorrect  超时响应,请注意!!
[2019-04-12 17:29:20,628] - log_print.py] - ERROR: 5 .天机api认证:http://116.62.67.80:9050/tjVerify/apiVerify  超时响应,请注意!!
[2019-04-12 17:29:30,065] - log_print.py] - ERROR: 6 .用户额度查询:http://116.62.67.80:9050/baseVerify/userAmount  超时响应,请注意!!
[2019-04-12 17:29:37,467] - log_print.py] - ERROR: 7 .手机通讯录上传:http://116.62.67.80:9050/baseVerify/addContacts  超时响应,请注意!!
[2019-04-12 17:29:45,151] - log_print.py] - ERROR: 8 .手机通话和短信记录:http://116.62.67.80:9050/baseVerify/addSelfContacts  超时响应,请注意!!
[2019-04-12 17:39:08,378] - log_print.py] - ERROR: 1 .实名认证:http://116.62.67.80:9050/baseVerify/faceAuthentication  超时响应,请注意!!
[2019-04-12 17:40:23,763] - log_print.py] - ERROR: 1 .实名认证:http://116.62.67.80:9050/baseVerify/faceAuthentication  超时响应,请注意!!
[2019-04-12 17:40:31,246] - log_print.py] - ERROR: 2 .芝麻分认证:http://116.62.67.80:9050/riskVerify/zhiMaAuth  超时响应,请注意!!
[2019-04-12 17:40:38,345] - log_print.py] - ERROR: 3 .运营商认证:http://116.62.67.80:9050/tjVerify/operatorVerify  超时响应,请注意!!
[2019-04-12 17:40:45,464] - log_print.py] - ERROR: 4 .身份证纠错:http://116.62.67.80:9050/baseVerify/faceErrorCorrect  超时响应,请注意!!
[2019-04-12 17:40:52,558] - log_print.py] - ERROR: 5 .天机api认证:http://116.62.67.80:9050/tjVerify/apiVerify  超时响应,请注意!!
[2019-04-12 17:40:59,576] - log_print.py] - ERROR: 6 .用户额度查询:http://116.62.67.80:9050/baseVerify/userAmount  超时响应,请注意!!
[2019-04-12 17:41:06,704] - log_print.py] - ERROR: 7 .手机通讯录上传:http://116.62.67.80:9050/baseVerify/addContacts  超时响应,请注意!!
[2019-04-12 17:41:13,759] - log_print.py] - ERROR: 8 .手机通话和短信记录:http://116.62.67.80:9050/baseVerify/addSelfContacts  超时响应,请注意!!
